typings/index.d.ts

```diff
@@ -3661,6 +3674,30 @@ declare module 'discord.js' {
     $device?: string;
   }
 
+  interface WidgetMember {
```

Comment:
I've made this typedef because these members aren't real, the id is just a number (1, 2, 3, etc...) and I don't think an new class is needed